engine/include/cubos/engine/physics/physics_velocity.hpp

```diff
@@ -0,0 +1,24 @@
+/// @file
+/// @brief Component @ref cubos::engine::PhysicsVelocity.
+/// @ingroup physics-plugin
+
+#pragma once
+
+#include <glm/vec3.hpp>
+
+namespace cubos::engine
+{
+    /// @brief Component which defines parameters of a particle.
+    /// @note Should be used with @ref PhysicsMass and @ref Position.
+    /// @ingroup physics-plugin
+    struct [[cubos::component("cubos/physics_velocity", VecStorage)]] PhysicsVelocity
+    {
+        glm::vec3 velocity;
+        glm::vec3 acceleration;
+        glm::vec3 totalForce;
+        glm::vec3 gravity;
+        float damping;
```

Comment:
I agree we don't need acceleration, at the moment I'm using that as gravity which is wrong. The gravity here is supposed to be an acceleration, and I was thinking of it as being specific for this object, like you could set a different gravity for each object. We could have maybe a global value if this isn't set but that was the idea. So if we go with this, how would we have a constant specific to an entity without it being on one of it's components? Or should we just have a global gravity force?

Reply:
If we wanted to have each entity with a different gravity value, I would prefer it to be on a different component. That way we could even have, for example, a global gravity resource `Gravity`, which you could override by adding it to a specific entity and setting it to another value. What do you think?

Reply:
I like that, it's better than what I originally thought. In the meantime, I also remembered we could just have a component which holds a modifier that we always use when applying the gravity. It could also have a bool to deactivate gravity, even though this can be done by setting the value to 0, but it would be more of a user friendly thing, for the editor too. What do you think?